Editor/Scripts/DCCIntegration/AutoServerCreation/AutoSetup.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Threading;
+using UnityEditor;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Unity.MeshSync.Editor {
+
+[InitializeOnLoad]
+internal static class AutoSetup {
+
+    private static Server m_server;
+    
+    static AutoSetup() {
+        // Create server
+        var settings = ServerSettings.defaultValue;
+        
+        settings.port = AutoSetupSettings.instance.Port;
+        
+        if (!Server.Start(ref settings, out m_server)) {
+            Debug.LogError("[MeshSync] Could not start Server");
```

Comment:
Maybe worth specifying "Could no start auto-setup server" to distinguist from the regular server that a user would manually use for syncing

Reply:
There is a window pop-up when this happens now, see comment in follow-up PR https://github.com/unity3d-jp/MeshSync/pull/774#discussion_r966978586